### PR TITLE
Updated Getting Started

### PR DESCRIPTION
**Pull request by marcusnaslund**

### CODE REVIEW

File: content/android/gettingstarted.md [hunk 1]
```diff
@@ -17,33 +18,33 @@ This section will describe how to integrate Vidhance by wrapping the camera driv
 
 # Setting up device
 ## Enabling USB debugging
-You need to have USB debugging enabled on your device:
+You need to have USB debugging enabled on your device.
+
+Note: The following steps are correct for Nexus 5, 6 and 6P. Other devices may look somewhat different:
+
+**TODO install adb**
```
Comment (marcusnaslund) [inline content/android/gettingstarted.md]: Should we mention this at all or just assume they have what's needed?

File: content/android/gettingstarted.md [hunk 2]
```diff
@@ -131,55 +139,60 @@ Make sure the correct VideoProcessor header for your HAL version is included in
...
-#### Option 1
-You can use the function *getAlignedWidth()* located in *vidhance/graphicbuffer/GraphicBufferWrapper.h* to check the values for your device:
+#### Option 1 (Recommended)
+**TODO: Otestad**
+The function will return the values and print the aligned widths to `logcat` under the log tag *GraphicBufferWrapper* (See [Using DDMS](#UsingDDMS) for instructions). Use this output to create a predefined array with values. Example for Nexus 6: **TODO: Anpassa för Nexus 6P**
 ```
 /* VidhanceProcessor.cpp */
-#include "../vidhance/graphicbuffer/GraphicBufferWrapper.h"
+
+/* Predefined aligned width in bytes for RGBA8888 GraphicBuffers on Nexus 6 */
```
Comment (marcusnaslund) [inline content/android/gettingstarted.md]: What about for 6P?

File: content/android/gettingstarted.md [hunk 3]
```diff
@@ -99,11 +107,11 @@ The result should be something like:
 ```
 Device version: 0x302
 ```
-where the most significant digit is the major version number and the two least significant digits are the minor version number. In the above example the original HAL is therefore of version 3.2. Note that Vidhance SDK currently only supports HAL version 3.0 and 3.2 but support for older versions will be available soon.
+where the most significant digit is the major version number and the two least significant digits are the minor version number. In the above example the original HAL is therefore of version 3.2. Note that Vidhance SDK currently only supports HAL version 3.0 and 3.2 but support for older versions will be available soon. **TODO stämmer detta? Verkar finnas redan*
```
Comment (marcusnaslund) [inline content/android/gettingstarted.md]: "available soon"? Utdaterat?

File: content/android/gettingstarted.md [hunk 4]
```diff
@@ -72,26 +70,36 @@ You will need ndk-build located in the Android-NDK to build the sources for your
...
+We recommend that you modify the example implementation for Nexus 6P to create a compatible version for your device.
+
 ## Android dependencies
-The camera wrapper depends on libraries found in the Android source tree. We have provided the needed headers and libraries for Nexus 6 which should be compatible with any 32-bit ARM-based device running Android 5.0.1. The headers can be found in the *include* folder and the libraries in the *libs* folder. If your device uses a different architecture or Android version you may need to replace these libraries with ones compatible with your device. Contact us if you need help with this matter.
+The camera wrapper depends on libraries found in the Android source tree. We have provided the needed headers and libraries for Nexus 6P which should be compatible with any 32-bit ARM-based device running Android 5.0.1. The headers can be found in the *include* folder and the libraries in the *libs* folder. **TODO: Var är de?* If your device uses a different architecture or Android version, you may need to replace these libraries with ones compatible with your device. Contact us if you need help with this matter.
```
Comment (marcusnaslund) [inline content/android/gettingstarted.md]: Where are they?

File: content/android/gettingstarted.md (hunk 1 above)
Comment (emilwestergren) [inline content/android/gettingstarted.md]: We should mention how to install `adb` and `fastboot`.

File: content/android/gettingstarted.md [hunk 5]
```diff
@@ -99,11 +107,11 @@ The result should be something like:
 ```
 Device version: 0x302
 ```
-where the most significant digit is the major version number and the two least significant digits are the minor version number. In the above example the original HAL is therefore of version 3.2. Note that Vidhance SDK currently only supports HAL version 3.0 and 3.2 but support for older versions will be available soon.
+where the most significant digit is the major version number and the two least significant digits are the minor version number. In the above example the original HAL is therefore of version 3.2. Note that Vidhance SDK currently only supports HAL version 3.0 and 3.2 but support for older versions will be available soon. **TODO stämmer detta? Verkar finnas redan**
```
Comment (marcusnaslund) [inline content/android/gettingstarted.md]: "available soon"? Outdated?

File: content/android/gettingstarted.md [hunk 6]
```diff
@@ -72,26 +70,36 @@ You will need ndk-build located in the Android-NDK to build the sources for your
...
 
 ```sh
 git clone https://github.com/vidhance/android-camera-wrapper
 ```
 
 ## Downloading Vidhance library
-Inside the nexus6 example folder you will find *download_vidhance.sh*. This script can be used to download the correct version of Vidhance for your device. You will need a key to be able to begin the download. Run the script and follow the instructions:
+In the `android-camera-wrapper/nexus6p` folder, you will find *setup.sh*. Give this file execution rights, and run it:
 ```sh
 chmod a+x download_vidhance.sh
```
Comment (emilwestergren) [inline content/android/gettingstarted.md]: This is not valid anymore. You only need to run *one* `.sh` script, `setup.sh`. Then it's all functions from there (download_vidhance, build, push, etc)

Comment (marcusnaslund) [inline content/android/gettingstarted.md]: Right, I've updated the text above, not the script text. Thanks.

File: content/android/gettingstarted.md (hunk 4 above)
Comment (emilwestergren) [inline content/android/gettingstarted.md]: Previously the user could run the `download_vidhance` script and enter a license key. That would download the vidhance binary and also the needed headers and libraries to build the wrapper. I guess we want it to work in the same way.

Comment (marcusnaslund) [inline content/android/gettingstarted.md]: But you're talking about the camera wrapper, so I assumed the includes and libs would be in the camera-wrapper repo, they're not.

File: content/android/gettingstarted.md (hunk 5 above)
Comment (emilwestergren) [inline content/android/gettingstarted.md]: We still don't fully support HAL 1.0, so this is still valid.

File: content/android/gettingstarted.md (hunk 4 above)
Comment (emilwestergren) [inline content/android/gettingstarted.md]: No, do you think they should be?

File: content/android/gettingstarted.md (hunk 2 above)
Comment (emilwestergren) [inline content/android/gettingstarted.md]: This should be updated to Nexus 6P.

File: content/android/gettingstarted.md (hunk 4 above)
Comment (marcusnaslund) [inline content/android/gettingstarted.md]: Not necessarily, I was just confused by how the text was formulated. I will add that they're downloaded along with Vidhance.

**Comment by emilwestergren** [comment]
Looks good as a first step. We need to fix a lot of other things but I think it's probably better if someone in the integration team does it.

**Comment by marcusnaslund** [comment]
Thanks. I will do some more tomorrow, then we can see if there's anything more I can help with.

**Comment by emilwestergren** [comment]
I can take over now if you want and let you review?

**Comment by marcusnaslund** [comment]
> I can take over now if you want and let you review?

Sure, you're the boss. Do we merge this and you make a new PR, or will you take my branch and make a new PR?

**Comment by emilwestergren** [comment]
You can merge this.

**Pull request merged**